src/components/upgrader/ipUpgrader.ts

```diff
@@ -438,7 +438,7 @@ export default class IpCameraUpgrader extends EventEmitter implements IDeviceUpg
         Logger.warn(errMsg, this.className);
         this.emit(CameraEvents.UPGRADE_FAILED, errMsg);
         reject(errMsg);
-      }, 10000);
+      }, 42000);
```

Comment:
We really need this long to read the cpio?

Reply:
Don't think so. Was told that this was used elsewhere, but honestly shouldn't take that long.